src/lib/y2storage/dialogs/proposal.rb

```diff
@@ -125,24 +125,90 @@ def propose!
       def summary
         # TODO: if there is a proposal, use the meaningful description with
         # hyperlinks instead of just delegating the summary to libstorage
-        content = if devicegraph
-          @actions_presenter.to_html
-        else
-          Yast::HTML.Para(
-            _(
-              "It was not possible to automatically propose a partitioning layout " \
-              "based on the current settings."
+        content = devicegraph ? actions_html : failure_html
+
+        RichText(Id(:summary), content)
+      end
+
+      # Text for the summary in cases in which a devicegraph was properly
+      # calculated
+      #
+      # @see #summary
+      #
+      # @return [String] HTML-formatted text
+      def actions_html
+        # Reuse the exact string "Changes to partitioning" from the partitioner
+        actions_source_html + _("<p>Changes to partitioning:</p>") + @actions_presenter.to_html
+      end
+
+      # @see #actions_html
+      def actions_source_html
+        return actions_source_for_partitioner unless proposal
+        return actions_source_for_guided_setup unless settings_adjustment
+        return actions_source_for_default_settings if settings_adjustment.empty?
+
+        para(_("Initial layout proposed after adjusting the Guided Setup settings:")) +
```

Comment:
the wording is a bit strange to me. Why it mention "adjusting the Guided Setup". If I get it, user does not run guided setup ( because if so, then second return is in effect, right? ), so it should be more like "Initial layout proposed after adjusting the Product settings:"

Reply:
To give the users a hint so they know for sure than that initial proposal uses the same logic and settings they can reach by clicking on "Guided Setup".

Reply:
But for me it looks like that installer think that user use guided setup. At least it is confusing for me.

Reply:
And "Product settings" is not the right wording either. The settings are actually defined by the combination of product, module/extension, installation role and whatnot.

Reply:
ok, so maybe "default settings" is better as it emphasize that it is default without any user modification, but with reduction needed by small disk.

Reply:
In other words, using "Guided Setup" was intentional here. As a way of telling the user than the initial proposal is just 100% equivalent to running the Guided Setup with the described settings.